src/core/Form/Checkbox/Checkbox.baseStyles.tsx

```diff
@@ -139,20 +136,15 @@ export const baseStyles = withSuomifiTheme(
       z-index: -9999;
     }
 
-    & .fi-checkbox_hintText {
-      display: block;
+    & .fi-hint-text {
       padding-left: ${theme.spacing.l};
-      ${theme.typography.bodyTextSmall};
       color: ${theme.colors.depthDark1};
+      margin-bottom: 0;
     }
 
-    & .fi-checkbox_status {
+    & .fi-status-text {
       display: block;
-      margin-top: ${theme.spacing.xxs};
-      color: ${theme.colors.blackBase};
-      font-size: 14px;
       line-height: 18px;
-      font-weight: 600;
     }
```

Comment:
Left the styles that was required to keep the visuals to left untouched.

Reply:
Is this different from other use cases?

Reply:
Or is this something that could / should be applied to status-text styles instead?

Reply:
That is good question, not sure which way is the most used way at the moment: which "wins".

These seems to be needed also when doing similar changes for _**Textarea**_ component.

**HintText**
` margin-bottom: 0;`

**StatusText**
`display: block;` and `line-height: 18px;`

Reply:
That's a good question. Optimally, only styles shared by all components should be in status/hintText. In my opinion this is also a design issue, not sure why we should have context specific styles for the same component. Another option, if possible, is to use the styles in the wrapping component instead and perhaps the components should not introduce any outside margins etc. and leave that to wrapping components?

Reply:
Decided to fix these with another issue / branch.